src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/plugins/_jquery-ui.scss

```diff
@@ -1,1559 +0,0 @@
-/*! jQuery UI - v1.11.3 - 2015-03-09
```

Comment:
was this the last place where we used jquiry-ui? because I see stuff like draggable, and I think we are still using that

Reply:
fixed b03c21d